config/v3_0_experimental/schema/ignition.json

```diff
@@ -73,6 +76,23 @@
             }
           }
         },
+        "proxy": {
+          "type": "object",
+          "properties": {
+            "httpProxy": {
+              "type": ["string", "null"]
```

Comment:
@ajeddeloh Is this what you wanted? I want to check before I fix the black box tests.

Reply:
Yup!